core/src/config.rs

```diff
@@ -103,3 +103,23 @@ fn admin_public_key_from(
 
     Ok(PublicKey::from(&SecretKey::from_bytes(&hash)?))
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[test]
+    fn admin_public_key() {
+        let email: String = "pj@aa.pl".to_string();
```

Comment:
```suggestion
        let email = "pj@aa.pl";
```

Reply:
Config is expecting [String struct](https://github.com/Holo-Host/holo-config/blob/2019-10-quickstart/core/src/config.rs#L65) as parameters rather than `&str` literal